Cargo.toml

```diff
@@ -1,8 +1,8 @@
 [workspace]
 
 members = [
+    "audit-info",
```

Comment:
thought this was going to be `auditable-info` ?

Reply:
We could still switch if you think that's better. I wanted a clean separation between the high-level crate and the low-level ones, but maybe this will just create confusion with the `rust-audit-info` binary.

Reply:
Yeah I think `auditable-info` is best around those considerations re: confusion.